test_util/integration_test.py

```diff
@@ -15,13 +18,19 @@
 import requests
 import retrying
 
+from ssh.ssh_tunnel import SSHTunnel, run_ssh_cmd
```

Comment:
Hmmm thanks. I changed it because `import ssh_tunnel` was also not working and complaining about import failures... @mellenburg any suggestions?